pkg/config/config_template.yaml

```diff
@@ -306,6 +306,30 @@ metadata_collectors:
 #
 {{ end -}}
 
+# Datadog Cluster Agent - DCA
```

Comment:
This block of documentation should live under a "context" (see https://github.com/DataDog/datadog-agent/blob/dd5c72fc063c1b09b60790e369ff03435bba73f0/pkg/config/render_config.go#L19-L34).

if it should be common to all builds (agent, dogstatsd and dca), please use `Common`, or feel free to add a more appropriate "context" name :)

Reply:
Also, can you add a quick explanation on when a user should care at all about the `Datadog Cluster Agent` settings?